### PR TITLE
Ux137 ng states

### PR DESCRIPTION
**Pull request by jeremie-lucca**
With demo & styleguide refactoring, this branch is generating a lot of conflicts. 
I think it would be a good idea to merge this part now. 
(It's about create mixin which could be use by ng classes)

### CODE REVIEW

File: packages/scss/src/mixins/_loading.scss [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+	&:before{
+		border: 2px solid rgba(0,0,0,.2);
+		border-radius: 50%;
+		bottom: 0;
+		content: '';
+		display: block;
+		height: $size;
+		left: 0;
+		line-height: .8rem;
```
Comment (sanlucca) [inline packages/scss/src/mixins/_loading.scss]: Doesn't line-height depend on $size too ?

Comment (jeremie-lucca) [inline packages/scss/src/mixins/_loading.scss]: I tried with 5 different button sizes and it was ok... :)

**Comment by lucienbertin** [comment]
@sanlucca @jeremie-lucca xan i merge this ?

**Comment by jeremie-lucca** [comment]
@lucienbertin Yep!

**Pull request merged**